src/features/PopularCitiesList.jsx

```diff
@@ -0,0 +1,29 @@
+import usePopularCities from '../hooks/usePopularCities';
+import CitiesList from '../components/common/CitiesList/CitiesList';
+
+const PopularCitiesList = ({ limit, increaseLimit }) => {
+  const {
+    data,
+    error,
+    isLoading,
+    fetchNextPage,
+    hasNextPage,
+    isFetchingNextPage,
+  } = usePopularCities();
+
+  return (
+    <CitiesList
+      limit={limit}
+      increaseLimit={increaseLimit}
```

Comment:
`CitiesList` accepts only following props `{ limit, data: cities = [], error, isLoading }`

Stop passing others not supported props

Reply:
Still not resolved...